host/windows/syscall.c

```diff
@@ -919,7 +921,7 @@ ssize_t oe_syscall_send_ocall(
     ssize_t ret;
     _set_errno(0);
 
-    ret = send(_get_socket(sockfd), buf, len, flags);
+    ret = send(_get_socket(sockfd), buf, (int)len, flags);
```

Comment:
This looks like it needs an arithmetic check that the downcast from unsigned 64-bit `size_t` to signed 32-bit `int` does not result in truncation.

Reply:
Yes, I added it because the compiler complained about this. I will see if I can add some checks before it.